components/org.wso2.carbon.identity.remotefetch.core/src/main/java/org/wso2/carbon/identity/remotefetch/core/impl/handlers/action/PollingActionListener.java

```diff
@@ -297,18 +299,30 @@ private void pollDirectory(ConfigDeployer deployer) {
                                     this.remoteFetchConfigurationId);
                     velocityTemplatedSPDeployer.deploy(configurationFileStream);
 
-                    deploymentRevision.setDeploymentStatus(DeploymentRevision.DeploymentStatus.DEPLOYED);
+                    deploymentRevision.setDeploymentStatus(DeploymentRevision.DeploymentStatus.SUCCEED);
                     deploymentRevision.setErrorMessage(null);
 
                 } catch (RemoteFetchCoreException | IOException e) {
                     log.error("Error Deploying " + sanitize(deploymentRevision.getFile().getName()), e);
-                    deploymentRevision.setDeploymentStatus(DeploymentRevision.DeploymentStatus.ERROR_DEPLOYING);
+                    deploymentRevision.setDeploymentStatus(DeploymentRevision.DeploymentStatus.FAILED);
                     deploymentRevision.setErrorMessage(RemoteFetchConfigurationUtils.trimErrorMessage(e.getMessage(),
                             e));
                 }
 
                 // Set new deployment Date
                 deploymentRevision.setDeployedDate(new Date());
+                // Set last iteration date as synced Date
+                deploymentRevision.setSyncedDate(this.lastIteration);
+
+                try {
+                    this.deploymentRevisionDAO.updateDeploymentRevision(deploymentRevision);
+                } catch (RemoteFetchCoreException e) {
+                    log.error("Error updating DeploymentRevision for " + sanitize(deploymentRevision.getItemName())
```

Comment:
log.error("Error updating DeploymentRevision for : " + sanitize(deploymentRevision.getItemName())
Please note the colon